api/role/schema.py

```diff
@@ -18,9 +21,17 @@ class Arguments:
     role = graphene.Field(Role)
 
     def mutate(self, info, **kwargs):
-        role = RoleModel(**kwargs)
-        with SaveContextManager(role, kwargs.get('role'), 'Role'):
-            return CreateRole(role=role)
+        try:
+            role = RoleModel(**kwargs)
+            query = Role.get_query(info)
+            role_name = query.filter(
+                func.lower(RoleModel.role) == func.lower(kwargs.get('role')))
+            if role_name.count() > 0:
```

Comment:
I think for this you can just have if role_name.count() and leave out the > 0 part.

Reply:
Done @caveinn